src/com/itacademy/eshop/services/ShopService.java

```diff
@@ -23,7 +25,14 @@ private ArrayList<Product> createProducts() {
          * one per each category in Category enum.
          * Product should not have any reviews.
          */
-        throw new UnsupportedOperationException("createProducts() is not implemented yet");
+        ArrayList<Product> arrayList = new ArrayList<>();
```

Comment:
name arrayList gives np context about the items in the collection. I would suggest adding some context to your name and have it like `products` which indicates that there is a collection of products or even  `productList`